core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -120,8 +120,9 @@ statement
     | CALL qualifiedName '(' (callArgument (',' callArgument)*)? ')'   #call
     | CREATE (OR REPLACE)? functionSpecification                       #createFunction
     | DROP FUNCTION (IF EXISTS)? functionDeclaration                   #dropFunction
-    | CREATE (OR REPLACE)? BRANCH (IF NOT EXISTS)? identifier
-        (WITH properties)? IN TABLE qualifiedName                      #createBranch
+    | CREATE (OR REPLACE)? BRANCH (IF NOT EXISTS)? branch=identifier
+        (WITH properties)? IN TABLE qualifiedName
+        (FROM from=identifier)?                                        #createBranch
```

Comment:
What does it actually mean when you replace a branch, but with a `from` branch specified, does the `from` will be ignored?